src/winetricks

```diff
@@ -14827,6 +14827,54 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata powershell apps \
+    title="Microsoft Powershell" \
+    publisher="Microsoft" \
+    year="2006" \
+    media="download" \
+    file1="ProtectionId.685.December.2016.rar" \
```

Comment:
Wrong file.

Reply:
Draft merge request -> Not expected to be reviewed untill finished.

Using draft so that i could get feedback for compatibility which makes the whole process easier.

Reply:
Ah i think that this is a feedback? sorry will fix the metadata last since i don't know what to fill in them untill `load_powershell` is working